appveyor.yml

```diff
@@ -1,13 +1,17 @@
 version: '{build}'
 
 environment:
-  mssql_jdbc_test_connection_properties: jdbc:sqlserver://localhost:1433;instanceName=SQL2016;databaseName=master;username=sa;password=Password12!;
+  JAVA_HOME: C:\Program Files\Java\jdk1.8.0 
+
   matrix:
-    - JAVA_HOME: C:\Program Files\Java\jdk1.8.0 
+    - mssql_jdbc_test_connection_properties: jdbc:sqlserver://localhost:1433;instanceName=SQL2008R2SP2;databaseName=master;username=sa;password=Password12!;
+
+    - mssql_jdbc_test_connection_properties: jdbc:sqlserver://localhost:1433;instanceName=SQL2016;databaseName=master;username=sa;password=Password12!;
 
 services:
+  - mssql2008r2sp2
```

Comment:
thank you @nsidhaye for sharing the link! I did find this link when I was doing investigation. I really love the idea to specify variables in matrix and then use CMD to start services! I updated my PR. Thank you for knowledge sharing!